tutorials/embedded/step-5.html.haml

```diff
@@ -28,7 +28,7 @@ title: Embedded Step-by-Step Tutorial
             :escaped
               GlobalConfigurationBuilder global = GlobalConfigurationBuilder.defaultClusteredBuilder();
               global.transport().clusterName("WeatherApp");
-              cacheManager = new DefaultCacheManager(global.build(), config.build());
```

Comment:
[Here](https://github.com/diegolovison/example-infinispan/blob/b7bf0faca266249761d9f381e1ee42581a40d986/src/main/java/com/github/diegolovison/example/infinispan/ClusteredCacheManagerExample.java#L27) is the example